### ISSUE TITLE
Fix Landing page app & RDM inconsistencies

### ISSUE THREAD

**Issue by zguillen** [MSD-LIVE/issues#158]
1. widen landing page app to match RDM, make sure all pages are same width (some currently are not) - change max width of page but edit MUI's theme typography element to limit it's width so that paragraphs don't span the entire page
2. drop down nav menus - some are centered some are not
3. page titles - some are centered some are not, make sure all are same H1 or whatever font size/style
4.

**Comment by zguillen** [comment]
@clansing @jugovimm can you think of other items we've noted recently that is inconsistent or needs to be changed?

**Comment by jugovimm** [comment]
@zguillen I'm getting a start on this ticket. Do we want the navigation between RDM and the landing page to agree as well? (ex. put "Acknowledgement" in the upper nav in RDM, resize the MSD-LIVE logo so the two match, etc.)

## RDM
![Image](https://github.com/MSD-LIVE/issues/assets/144269927/f36165eb-f12b-41aa-8f2f-1da330a1389f)

## Landing Page
![Image](https://github.com/MSD-LIVE/issues/assets/144269927/23c0dded-b24d-4305-8600-d6bf0d02eed8)

**Comment by jugovimm** [comment]
@zguillen Also, some of the menus have different links. Is this something to fix as well? I suppose I'm wondering just how _**comprehensive**_ this refactor to match the inconsistencies between RDM and the landing page should be

## RDM Help Dropdown
![Image](https://github.com/MSD-LIVE/issues/assets/144269927/80a1100d-3822-4382-8406-4d12c6970f8c)

## Landing Page Help Dropdown
![Image](https://github.com/MSD-LIVE/issues/assets/144269927/2a0e9932-019b-427c-9e02-c8040782c60a)

**Comment by zguillen** [comment]
yes please make the RDM drop down match the landing page's

**Comment by zguillen** [comment]
yes, please have the top level nav match, don't worry about same exact sizes for logos, font's etc.  Just make sure the words and links match. Also, have RDM match landing page
